cloud/amazon/rds_snapshot_copy.py

```diff
@@ -0,0 +1,155 @@
+      - cannot be empty
+      - must contain 1-255 alphanumeric characters or hyphens
+      - first character must be a letter
+      - cannot end with a hypen or contain two consecutive hyphens
+    required: true
+
+author: "Brock Haywood (@brockhaywood)"
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+
+'''
+
+try:
+    import boto.rds
+    HAS_BOTO = True
+except ImportError:
+    HAS_BOTO = False
+
+
+def copy_snapshot(module, source_conn, conn,
+                  source_db_snapshot_identifier=None,
+                  target_db_snapshot_identifier=None):
+
+    if not source_db_snapshot_identifier or not target_db_snapshot_identifier:
+        module.fail_json(msg='Source DB Snapshot Id and Target DB Snapshot '
+                             'Id must be provided')
+
+    try:
+        snapshots = conn.get_all_dbsnapshots(
+            snapshot_id=source_db_snapshot_identifier)
+    except boto.exception.BotoServerError, e:
+        module.fail_json(msg = "%s: %s" % (e.error_code, e.error_message))
+
+    if not snapshots:
+        module.fail_json(msg="Could not find snapshot with "
+                             "identifier %s" % (source_db_snapshot_identifier))
+
+    snapshot = snapshots[0]
+
+    if snapshot.status != 'available':
+        module.fail_json(msg="Snapshot not available with identifier "
+                             "%s" % (source_db_snapshot_identifier))
+
+    snapshot = conn.copy_dbsnapshot(
+        source_snapshot_id=source_db_snapshot_identifier,
+        target_snapshot_id=target_db_snapshot_identifier
+    )
+
+    module.exit_json(changed=True,
+                     snapshot_id=snapshot.id)
+
+
+def copy_rds_snapshot_ansible_module():
+    argument_spec = ec2_argument_spec()
+    argument_spec.update(
+        dict(
+            region = dict(),
+            source_region = dict(),
+            source_db_snapshot_identifier = dict(),
+            target_db_snapshot_identifier = dict(),
+        )
+    )
+    module = AnsibleModule(argument_spec=argument_spec)
+    return module
+
+
+def main():
+    module = copy_rds_snapshot_ansible_module()
+
+    if not HAS_BOTO:
+        module.fail_json(msg='boto required for this module')
+
+    source_db_snapshot_identifier = module.params.get(
+        'source_db_snapshot_identifier')
+    source_region = module.params.get('source_region')
+    target_db_snapshot_identifier = module.params.get(
+        'target_db_snapshot_identifier')
+
+    # Retrieve any AWS settings from the environment.
+    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module)
+
+    if not region:
+        module.fail_json(msg = str("Either region or AWS_REGION or EC2_REGION "
+                                   "environment variable or boto config "
+                                   "aws_region or ec2_region must be set."))
+
+    try:
+        conn = boto.rds.connect_to_region(region, **aws_connect_kwargs)
```

Comment:
please use `connect_to_aws(boto.rds, region, **aws_connect_kwargs)` 

(see ansible/ansible-modules-core#2421 for details)